### PR TITLE
bugfix: fix remove mesages

### PR DESCRIPTION
**Pull request by rprzedzik**
# Description

Fix message show when given element can be removed from system. 

Issue 
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Unit Tests
- [ ] Behat Test

# Checklist:
- [ ] I have read the contribution requirements and fulfill them.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: module/attribute/src/Infrastructure/Strategy/Relationship/AttributeAttributeGroupRelationshipStrategy.php [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 class AttributeAttributeGroupRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with attribute %relations%';
+    private const ONE_MESSAGE = 'Attribute group has a relation with attribute';
```
Comment (ochowa) [inline module/attribute/src/Infrastructure/Strategy/Relationship/AttributeAttributeGroupRelationshipStrategy.php:21]: with an attribute

File: module/attribute/src/Infrastructure/Strategy/Relationship/AttributeUnitRelationshipStrategy.php [hunk 2]
```diff
@@ -18,7 +18,8 @@
 
 class AttributeUnitRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with attribute %relations%';
+    private const ONE_MESSAGE = 'Unit group has a relation with attribute';
```
Comment (ochowa) [inline module/attribute/src/Infrastructure/Strategy/Relationship/AttributeUnitRelationshipStrategy.php:21]: an attribute

File: module/category/src/Infrastructure/Strategy/Relationship/CategoryInCategoryTreeRelationshipStrategy.php [hunk 3]
```diff
@@ -18,7 +18,8 @@
 
 class CategoryInCategoryTreeRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Category has relationships with category tree';
+    private const ONE_MESSAGE = 'Category has a relation with category tree';
```
Comment (ochowa) [inline module/category/src/Infrastructure/Strategy/Relationship/CategoryInCategoryTreeRelationshipStrategy.php:21]: a category tree

File: module/designer/src/Infrastructure/Strategy/Relationship/ProductTemplateRelationshipStrategy.php [hunk 4]
```diff
@@ -18,7 +18,8 @@
 
 class ProductTemplateRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Template has a relation with product';
```
Comment (ochowa) [inline module/designer/src/Infrastructure/Strategy/Relationship/ProductTemplateRelationshipStrategy.php:21]: a product

File: module/designer/src/Infrastructure/Strategy/Relationship/TemplateAttributeRelationshipStrategy.php [hunk 5]
```diff
@@ -18,7 +18,8 @@
 
 class TemplateAttributeRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with template %relations%';
+    private const ONE_MESSAGE = 'Attribute has a relation with template';
```
Comment (ochowa) [inline module/designer/src/Infrastructure/Strategy/Relationship/TemplateAttributeRelationshipStrategy.php:21]: a template

File: module/designer/src/Infrastructure/Strategy/Relationship/TemplateMultimediaRelationshipStrategy.php [hunk 6]
```diff
@@ -18,7 +18,8 @@
 
 class TemplateMultimediaRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with multimedia %relations%';
+    private const ONE_MESSAGE = 'Multimedia has a relation with template';
```
Comment (ochowa) [inline module/designer/src/Infrastructure/Strategy/Relationship/TemplateMultimediaRelationshipStrategy.php:21]: have a relation with a template. 

Multimedia is plural

File: module/designer/src/Infrastructure/Strategy/Relationship/TemplateMultimediaRelationshipStrategy.php [hunk 7]
```diff
@@ -18,7 +18,8 @@
...
+    private const MULTIPLE_MESSAGE = 'Multimedia has %count% relations with some templates';
```
Comment (ochowa) [inline module/designer/src/Infrastructure/Strategy/Relationship/TemplateMultimediaRelationshipStrategy.php:22]: have

File: module/exporter-file/src/Infrastructure/Strategy/Relationship/LanguageErgonodeChannelRelationshipStrategy.php [hunk 8]
```diff
@@ -19,7 +19,8 @@
 
 class LanguageErgonodeChannelRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with channel %relations%';
+    private const ONE_MESSAGE = 'Language is used in channel';
```
Comment (ochowa) [inline module/exporter-file/src/Infrastructure/Strategy/Relationship/LanguageErgonodeChannelRelationshipStrategy.php:22]: a channel

File: module/product-collection/src/Infrastructure/Relationship/ProductProductCollectionRelationshipStrategy.php [hunk 9]
```diff
@@ -18,7 +18,8 @@
 
 class ProductProductCollectionRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product collection %relations%';
+    private const ONE_MESSAGE = 'Product has a relation with collection';
```
Comment (ochowa) [inline module/product-collection/src/Infrastructure/Relationship/ProductProductCollectionRelationshipStrategy.php:21]: a collection

File: module/product-collection/src/Infrastructure/Strategy/Relationship/ProductCollectionProductCollectionTypeRelationshipStrategy.php [hunk 10]
```diff
@@ -18,7 +18,8 @@
 
 class ProductCollectionProductCollectionTypeRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with collection %relations%';
+    private const ONE_MESSAGE = 'Collection Type has a relation with collection';
```
Comment (ochowa) [inline module/product-collection/src/Infrastructure/Strategy/Relationship/ProductCollectionProductCollectionTypeRelationshipStrategy.php:21]: a collection

File: module/product/src/Infrastructure/Strategy/Relationship/ChildrenProductRelationshipStrategy.php [hunk 11]
```diff
@@ -18,7 +18,8 @@
 
 class ChildrenProductRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Product has a relation with another product';
+    private const MULTIPLE_MESSAGE = 'Product has %count% relations with others products';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ChildrenProductRelationshipStrategy.php:22]: other products

File: module/product/src/Infrastructure/Strategy/Relationship/ProductAttributeRelationshipStrategy.php [hunk 12]
```diff
@@ -18,7 +18,8 @@
 
 class ProductAttributeRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Attribute has a relation with product';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductAttributeRelationshipStrategy.php:21]: a product

File: module/product/src/Infrastructure/Strategy/Relationship/ProductCategoryRelationshipStrategy.php [hunk 13]
```diff
@@ -18,7 +18,8 @@
 
 class ProductCategoryRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Category has a relation with product';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductCategoryRelationshipStrategy.php:21]: a product

File: module/product/src/Infrastructure/Strategy/Relationship/ProductMultimediaRelationshipStrategy.php [hunk 14]
```diff
@@ -18,7 +18,8 @@
 
 class ProductMultimediaRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with multimedia %relations%';
+    private const ONE_MESSAGE = 'Multimedia has a relation with product';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductMultimediaRelationshipStrategy.php:21]: have a relation with a product

File: module/product/src/Infrastructure/Strategy/Relationship/ProductMultimediaRelationshipStrategy.php [hunk 15]
```diff
@@ -18,7 +18,8 @@
...
+    private const MULTIPLE_MESSAGE = 'Multimedia has %count% relations with some products';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductMultimediaRelationshipStrategy.php:22]: multimedia have

File: module/product/src/Infrastructure/Strategy/Relationship/ProductOptionRelationshipStrategy.php [hunk 16]
```diff
@@ -17,7 +17,8 @@
 
 class ProductOptionRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Option has a relation with product %sku%';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductOptionRelationshipStrategy.php:20]: with a product
+ remove the variable?

File: module/product/src/Infrastructure/Strategy/Relationship/ProductRelationAttributeRelationshipStrategy.php [hunk 17]
```diff
@@ -18,7 +18,8 @@
 
 class ProductRelationAttributeRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Product has a relation with another product';
+    private const MULTIPLE_MESSAGE = 'Product has %count% relations with others products';
```
Comment (ochowa) [inline module/product/src/Infrastructure/Strategy/Relationship/ProductRelationAttributeRelationshipStrategy.php:22]: other

File: module/segment/src/Infrastructure/Strategy/Relationship/ConditionSetSegmentRelationshipStrategy.php [hunk 18]
```diff
@@ -18,7 +18,8 @@
 
 class ConditionSetSegmentRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with segment %relations%';
+    private const ONE_MESSAGE = 'Condition set has a relation with segment';
```
Comment (ochowa) [inline module/segment/src/Infrastructure/Strategy/Relationship/ConditionSetSegmentRelationshipStrategy.php:21]: a segment

File: module/workflow/src/Infrastructure/Strategy/Relationship/ConditionSetWorkflowTransitionRelationshipStrategy.php [hunk 19]
```diff
@@ -17,7 +17,8 @@
 
 class ConditionSetWorkflowTransitionRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with workflow transition %relations%';
+    private const ONE_MESSAGE = 'Condition set has a relation with transition';
```
Comment (ochowa) [inline module/workflow/src/Infrastructure/Strategy/Relationship/ConditionSetWorkflowTransitionRelationshipStrategy.php:20]: a status transition

File: module/workflow/src/Infrastructure/Strategy/Relationship/ConditionSetWorkflowTransitionRelationshipStrategy.php [hunk 20]
```diff
@@ -17,7 +17,8 @@
...
+    private const MULTIPLE_MESSAGE = 'Condition set has %count% relations with some transitions';
```
Comment (ochowa) [inline module/workflow/src/Infrastructure/Strategy/Relationship/ConditionSetWorkflowTransitionRelationshipStrategy.php:21]: with some status transitions

File: module/workflow/src/Infrastructure/Strategy/Relationship/DefaultStatusRelationshipStrategy.php [hunk 21]
```diff
@@ -18,7 +18,8 @@
 
 class DefaultStatusRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with workflow {relations}';
+    private const ONE_MESSAGE = 'Status has a relation with workflow';
```
Comment (ochowa) [inline module/workflow/src/Infrastructure/Strategy/Relationship/DefaultStatusRelationshipStrategy.php:21]: a workflow

File: module/workflow/src/Infrastructure/Strategy/Relationship/StatusProductRelationshipStrategy.php [hunk 22]
```diff
@@ -18,7 +18,8 @@
 
 class StatusProductRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with product %relations%';
+    private const ONE_MESSAGE = 'Status has a relation with product';
```
Comment (ochowa) [inline module/workflow/src/Infrastructure/Strategy/Relationship/StatusProductRelationshipStrategy.php:21]: a product

File: module/workflow/src/Infrastructure/Strategy/Relationship/StatusWorkflowRelationshipStrategy.php [hunk 23]
```diff
@@ -19,7 +19,8 @@
 
 class StatusWorkflowRelationshipStrategy implements RelationshipStrategyInterface
 {
-    private const MESSAGE = 'Object has active relationships with workflow {relations}';
+    private const ONE_MESSAGE = 'Status has a relation with workflow';
```
Comment (ochowa) [inline module/workflow/src/Infrastructure/Strategy/Relationship/StatusWorkflowRelationshipStrategy.php:22]: a workflow

**Review by ochowa — CHANGES_REQUESTED** [review]

**Review by ochowa — APPROVED** [review]

**Review by wfajczyk — APPROVED** [review]

**Review by bleto — APPROVED** [review]

**Pull request merged**